mempool/clist_mempool.go

```diff
@@ -29,7 +29,7 @@ import (
 // mempool uses a concurrent list structure for storing transactions that can
 // be efficiently accessed by multiple concurrent readers.
 type CListMempool struct {
-	config *cfg.MempoolConfig
+	config *cfg.Config
```

Comment:
we must not require the full config here

Reply:
we need to have timeout for mempool before sending notification. I think commit-timeout is fair enough. 
I didn't want to add new item to the mempool config. and I used the consensus config. It makes sense.

Reply:
In my tests I never faced this condition.